helm/alfresco-content-services/requirements.yaml

```diff
@@ -21,3 +21,10 @@ dependencies:
   version: 1.2.0
   condition: alfresco-digital-workspace.enabled
   repository: https://kubernetes-charts.alfresco.com/stable
+- name: alfresco-sync-service
+  version: 1.0.0-RC1
```

Comment:
It is not directly connected to this PR, but it seems odd to have a stable chart with RC in it.

Reply:
Thanks for the observation. I considered it stable because the work on the chart is considered done. It contains the RC in it because it references an RC version of Sync service image. 
I went with my gut feeling, I'm not aware of any Alfresco policies regarding Helm chart naming styles for stable charts.